internal/pkg/cmd/flags.go

```
@@ -389,3 +402,39 @@ func AddValueFormatFlag(cmd *cobra.Command) {
 		return arr
 	})
 }
+
+func AddLinkFlag(cmd *cobra.Command, command *AuthenticatedCLICommand) {
+	cmd.Flags().String("link", "", "Cluster link name.")
+
+	RegisterFlagCompletionFunc(cmd, "link", func(cmd *cobra.Command, args []string) []string {
+		if err := command.PersistentPreRunE(cmd, args); err != nil {
+			return nil
+		}
+
+		return AutocompleteLinks(command)
+	})
+}
+
+func AutocompleteLinks(command *AuthenticatedCLICommand) []string {
+	kafkaREST, err := command.GetKafkaREST()
```

FYI, you're ignoring this error.